fiftyone/plugins/skills.py

```
@@ -0,0 +1,253 @@
+
+    if category is not None:
+        category = {category} if isinstance(category, str) else set(category)
+
+    skills = []
+    for pd in plugin_defs:
+        try:
+            for skill in _iter_plugin_skills(pd):
+                if category is not None and skill.category not in category:
+                    continue
+                skills.append(skill)
+        except:
```

Ruff (0.15.7)

[error] Do not use bare `except`

(E722)